SimTracker/TrackerHitAssociation/src/TrackerHitAssociator.cc

```diff
@@ -114,7 +114,7 @@ void TrackerHitAssociator::makeMaps(const edm::Event& theEvent, const TrackerHit
           theEvent.labelsForToken(cfToken, labels);
           std::string trackerContainer(labels.productInstance);
```

Comment:
@makortel Thanks, I have overlooked that label only depends from cfToken.

I will now test if the change of the <code>if</code> position has any effect.
Thanky for the suggestions.